src/objects.js

```diff
@@ -1,41 +1,50 @@
 const createPerson = (name, age) => {
-  // your code here
+  this.name = name;
+  this.age = age;
+  return { name, age };
```

Comment:
Using the `this` keyword here doesnt make any sense, as you arent using the variables you just assigned in a JS object style way in the return statement. You could re-write this just to:
```
  return {
    name,
    age
  };
```